internal/namespaces/k8s/v1beta4/custom_node.go

```diff
@@ -0,0 +1,57 @@
+package k8s
+
+import (
+	"context"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/scaleway/scaleway-cli/internal/core"
+	"github.com/scaleway/scaleway-cli/internal/human"
+	k8s "github.com/scaleway/scaleway-sdk-go/api/k8s/v1beta4"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+const (
+	nodeActionTimeout = 10 * time.Minute
```

Comment:
we're starting to have a lot of different timeouts in different files, although they are used for the same kind of operations

maybe we should have a single file with base timeouts as `baseTimeout = x * unit`, and then reuse them in the different `custom_xxx.go` files with `nodeActionTimeout = baseTimeout` ?

Reply:
There are used for the same kind of operations, but not the same operations. In the future they may have different values. I can set a basetimeout in the meantime though

Reply:
I'll open a new PR to set a basetimeout once this one is merged if you don't mind :)